src/vis/bar/utils.ts

```diff
@@ -104,12 +208,16 @@ async function setPlotsWithGroupsAndMultiples(
 async function setPlotsWithGroups(
   columns: VisColumn[],
   catCol: VisCategoricalColumn,
+  aggType: EAggregateTypes,
+  aggColumn: VisNumericalColumn,
```

Comment:
```suggestion
  aggregateType: EAggregateTypes,
  aggregateColumn: VisNumericalColumn,
```

Name equal parameters equal also across functions.